src/pages/library/Datasets.js

```diff
@@ -240,6 +243,39 @@ const amppd = () =>
     ]
   );
 
+const axin = () =>
+  h(
+    Participant,
+    {
+      logo: { src: axinLogo, alt: 'AnalytiXIN logo', height: '55%' },
+      title: 'AnalytiX Indiana Health Dataset',
+      description: h(Fragment, [
+        'The ',
+        h(Link, { href: 'https://analytixindiana.com', ...Utils.newTabLinkProps }, 'AnalytiXIN life sciences health data asset (HDA)'),
+        ` has been designed through a working
+        collaboration involving Eli Lilly and Company, IU Health, the Indiana Biobank within the
+        IU School of Medicine, the Indiana Health Information Exchange, and other partners to build
+        a shared health-data platform linking consented clinical and genomic patient data. This
+        collaboration is further leveraging the Broad Institute’s technical and genomic
+        infrastructure to accelerate the build-out as well as connect to broader national
+        initiatives such as the National Institutes of Health All of Us research program.`,
+      ]),
+      sizeText: 'Participants: > 35,000',
+    },
+    [
+      h(
+        ButtonPrimary,
+        {
+          'aria-label': 'View AnalytiXIN details',
+          tooltip: 'View details about this dataset',
+          href: '/#library/builder/c3eb4708-444f-4cbf-a32c-0d3bb93d4819',
```

Comment:
I feel like this will require some maintenance to keep it up to date, even if it isn't frequent. Is there a way we could fetch the snapshot id somehow? Maybe not for this PR, but we could add a flag to indicate if a snapshot is the current release and find it that way?

Reply:
I agree, this should be introspected using a TDR API to find the correct ID. Although in practice it won't change often and once released to prod, it won't change at all for a few months.

I've changed this to store the information in `brand` which I think is better than hardcoding it here